sigs.yaml

```diff
@@ -1010,6 +1026,9 @@ sigs:
     - name: metrics-server
       owners:
       - https://raw.githubusercontent.com/kubernetes-incubator/metrics-server/master/OWNERS
+    - name: metrics
```

Comment:
Would like to hear from @brancz or @piosz to confirm this metrics repo should belong to sig-instrumentation

Reply:
Yep, this change correct :slightly_smiling_face:. sig-instrumentation has owned this all along, I'm not sure why this wasn't in here to begin with.